### PR TITLE
Add Next steps links to Start topics

### PR DESCRIPTION
**Pull request by beckykd**
Part of #464

### CODE REVIEW

File: docs/start/configure-qiskit-local.mdx [hunk 1]
```diff
@@ -52,6 +52,9 @@ Set these environment variables to alter the default behavior of Qiskit:
 
 ## Next steps
 
-<Admonition type="tip" title="Recommendation">
-    Follow the steps in [Hello world](hello-world) to write and run a quantum program.
+<Admonition type="tip" title="Recommendations">
+  - Read the [contributing guidelines](https://github.com/Qiskit/qiskit/blob/main/CONTRIBUTING.md) to contribute to the open-source Qiskit SDK.
```
Comment (Eric-Arellano) [inline docs/start/configure-qiskit-local.mdx:56]: Should this be lower in the list? Or maybe less strong, like 

> If interested in contributing, read the contributing guidelines....

We don't want to imply you need to contribute.

**Review by Eric-Arellano — COMMENTED** [review]
Thanks!

File: docs/start/configure-qiskit-local.mdx (hunk 1 above)
Comment (abbycross) [reply docs/start/configure-qiskit-local.mdx:56]: or maybe 

> - Read the [contributing guidelines](https://github.com/Qiskit/qiskit/blob/main/CONTRIBUTING.md) if you want to contribute to the open-source Qiskit SDK.
?

File: docs/start/setup-channel.mdx [hunk 2]
```diff
@@ -181,4 +181,6 @@ You need access credentials to use cloud-based IBM Quantum systems.
 <Admonition type="tip" title="Recommendations">
   -  [Configure Qiskit locally.](configure-qiskit-local)
   -  Follow the steps in [Hello world](hello-world) to write and run a quantum program.
+  -  [Get started with IBM Quantum on Cloud.](https://test.cloud.ibm.com/docs-draft/quantum-computing?topic=quantum-computing-get-started)
```
Comment (abbycross) [inline docs/start/setup-channel.mdx:184]: Looks like this is a staging link here?
```suggestion
  -  [Get started with IBM Quantum on Cloud.](https://cloud.ibm.com/docs/quantum-computing?topic=quantum-computing-get-started)
```

**Review by abbycross — APPROVED** [review]

File: docs/start/configure-qiskit-local.mdx [hunk 3]
```diff
@@ -52,6 +52,9 @@ Set these environment variables to alter the default behavior of Qiskit:
...
+  - Learn how to [build circuits](../build/).
+  - [Run the Hello world program](hello-world).
+  - Try a tutorial, such as [Grover's agorithm](https://learning.quantum.ibm.com/tutorial/grovers-algorithm).
```
Comment (abbycross) [inline docs/start/configure-qiskit-local.mdx:59]:
```suggestion
  - Try a tutorial, such as [Grover's algorithm](https://learning.quantum.ibm.com/tutorial/grovers-algorithm).
```

File: docs/start/configure-qiskit-local.mdx (hunk 1 above)
Comment (beckykd) [reply docs/start/configure-qiskit-local.mdx:56]: I'll do both - thanks for the input!

File: docs/start/setup-channel.mdx (hunk 2 above)
Comment (beckykd) [reply docs/start/setup-channel.mdx:184]: oh gosh - thanks!

**Pull request merged**